src/components/03-containers/three-column-container/_three-column-container.scss

```diff
@@ -0,0 +1,26 @@
+.sfgov-container-three-column {
+  flex-direction: column;
+  display: flex;
+  flex-wrap: wrap;
+  @include media($bp-narrow) {
+    flex-direction: row;
+  }
+  > div {
+    width: 100%;
+    margin-bottom: 10px;
```

Comment:
Since most of our components have `10px` multiples for spacing and sizing, maybe we should turn that into a `$rhythm` variable (using `rem`s) to embed that into the system?

Examples:

`margin-bottom: $rhythm;`
`padding: $rhythm * 2;`